pkg/cmd/issue/edit/edit_test.go

```diff
@@ -431,6 +438,27 @@ func mockRepoMetadata(_ *testing.T, reg *httpmock.Registry) {
 			"pageInfo": { "hasNextPage": false }
 		} } } }
 		`))
+	reg.Register(
```

Comment:
:thought_balloon: Tests for editing are pretty shallow and I'd like to improve them, but I am a bit lost how to approach them and add more. I am open for some advice on how to approach testing and set those up. Currently, I make sure existing tests are not broken and that the logic flow is correct and proper APIs are called. But testing the interactive mode more properly or different combinations would be nice